tools/starman-development.psgi

```diff
@@ -57,3 +62,16 @@ builder {
 };
 
 # -*- perl-mode -*-
+
```

Comment:
merging the entire PR, but I think this function can be removed if we can reduce the list of directories being monitored.